README.md

```diff
@@ -339,11 +340,20 @@ This repository provides the following examples of how to set up this plugin:
 
 ### Main tasks
 
-This plugin adds two maintasks to every source set: `ktlint[source set name]SourceSetCheck` and `ktlint[source set name]SourceSetFormat`.
-Additionally, a simple `ktlintCheck` task has also been added that checks all of the source sets for that project.
-Similarly, a `ktlintFormat` task has been added that formats all of the source sets.
-
-Additionally plugin adds two task for project kotlin script files: `ktlintKotlinScriptCheck` and `ktlintKotlinScriptFormat`.
+This plugin adds following tasks to every project:
+- `loadKtlintReporters` - preloads KtLint `Reporter`s
+- `runKtlintCheckOverKotlinScripts` - runs actual lint check over project Kotlin script files
+- `ktlintKotlinScriptCheck` - generates reports and prints issues into Gradle console based on lint check found errors
+- `runKtlintFormatOverKotlinScripts` - tries to format according to the code style project Kotlin script files
+- `ktlintKotlinScriptFormat` - generate reports and prints issues into Gradle console based on found unformattable errors
```

Comment:
It still actually attempts to make the formatting change, right? That's not clear here.

Reply:
`ktlintKotlinScriptFormat` depends on `runKtlintFormatOverKotlinScripts` task, so yes, it attempts to make the formatting change.

Will clarify it :+1:

Reply:
Fixed via e4c4017a2